src/core/Env.js

```diff
@@ -193,6 +211,10 @@ getJasmineRequireObj().Env = function(j$) {
       }
     };
 
+    this.getPromise = function() {
+      return config.promiseLibrary;
+    };
+
```

Comment:
This implementation of `getPromise` will only return the _configured_ library, and will never try to load up a default one already present in the execution environment (i.e. `window.Promise` or `global.Promise`).